bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -60,6 +60,7 @@
 				<option value="DD51">Dolby Digital 5.1</option>
 				<option value="DDPlus51">Dolby Digital Plus 5.1</option>
 				<option value="PCM51">Dolby Multichannel PCM 5.1</option>
+				<option value="DTS51">Dolby Surround 5.1</option>
```

Comment:
DTS is not Dolby.

Reply:
You're correct, brain fart. I meant to type DTS.

Reply:
Fixed in the most recent commit.